File: Assets/Fruit Cut Ninja Starter Kit/Scripts/PrepareLevel.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;
using UnityEngine.UI;

public class PrepareLevel : MonoBehaviour
{
    public GameObject GetReady;
    public GameObject GO;
    // Start is called before the first frame update
    private void Awake()
    {
        GetComponent<Timer>().timeAvailable = SharedSetting.ConfigTime;

    }
    void Start()
    {
        GameObject.Find("Canvas/LevelName/LevelName").GetComponent<Text>().text = SharedSetting.LevelName[SharedSetting.LoadLevel];
        StartCoroutine(PrepareRoutine());
    }
    IEnumerator PrepareRoutine()
    {
        yield return new WaitForSeconds(1.0f);
        GetReady.SetActive(true);
        yield return new WaitForSeconds(2.0f);
        GetReady.SetActive(false);
        GO.SetActive(true);
        yield return new WaitForSeconds(1.0f);
        GO.SetActive(false);
    }
    // Update is called once per frame
    void Update()
    {
        
    }
}

File: Assets/Fruit Cut Ninja Starter Kit/Scripts/PrepareLevel.js
//show couting down before game starts

#pragma strict
var GetReady : GameObject;
var GO : GameObject;
var started : boolean;
var fpsgo : GameObject;

function Awake() {
	//Set game time
	GetComponent(Timer).timeAvailable = SharedSettings.ConfigTime;
}

function PrepareRoutine()
{
yield WaitForSeconds(1.0);
GetReady.active = true;
yield WaitForSeconds(2.0);
GetReady.active = false;
GO.active = true;
GetComponent(Timer).RunTimer();
started = true;
yield WaitForSeconds(1.0);
GO.active = false;
}

function Start () {

	GameObject.Find("GUI/LevelName").GetComponent(GUIText).text = SharedSettings.LevelName[SharedSettings.LoadLevel-1]; 

	PrepareRoutine();
}


